google/apistore.go

```diff
@@ -59,3 +59,29 @@ func (c *APIStore) AddReader(bucket, object, entity string) error {
 	_, err := c.service.ObjectAccessControls.Insert(bucket, object, ac).Do()
 	return err
 }
+
```

Comment:
@bpopadiuk are these meant to be hidden from the `Store` interface?    Is the idea that the caller will have to type case the type to use these methods?

Reply:
@epsniff yeah, wasn't intending to change the interface, i've seen this type instantiated directly, so that was the idea via https://github.com/lytics/cloudstorage/pull/85/files#diff-4aabffc115f91176ac3ba490b942a66e34e63d981e386c15ef2f96bb9d22bceaL15

Reply:
@epsniff just checking in on this one, any concern with this? just adding these for consumers of this concrete type

Reply:
@epsniff any concern with merging this one?